test/runtests.jl

```diff
@@ -600,6 +600,21 @@ elseif instance_type == "ECS"
     end
 end
 
+@testset "Non-Amazon S3" begin
+  aws_google = AWSCore.aws_config(creds=nothing, region="", service_host="googleapis.com", service_name="storage")
+  huss_prefix = "CMIP6_3hr/IPSL-CM6A-LR/historical/r6i1p1f1/huss/"
+  result = AWSCore.Services.s3(aws_google, "GET", "/{Bucket}", Dict(:Bucket=>"pangeo-data",:prefix=>huss_prefix))
```

Comment:
It would probably be better to update this with the google-owned data in the `cmip6` bucket (instead of `pangeo-data`). 

Full file listing is here: https://pangeo-data.github.io/pangeo-datastore/cmip6_pangeo.html

Reply:
Ok, thanks for the link. I updated the test accordingly